EIPS/eip-1620.md

```diff
@@ -1,305 +1,198 @@
----
+```
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg) <hello@paulrberg.com>
+author: Paul Razvan Berg (@PaulRBerg) <paul@sablier.finance>
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Final
```

Comment:
One cannot just set an EIP to Final from Draft. It must go through the Last Call process. Please check EIP-1.

Reply:
Epic fail on my end. I updated the status to "Last Call" in September, but I forgot to submit a PR for that.